src/components/publishing/sections/text.tsx

```diff
@@ -1,8 +1,132 @@
-import styled from "styled-components"
+import React from "react"
+import styled, { StyledFunction } from "styled-components"
+import { pMedia } from "../../helpers"
 import Fonts from "../fonts"
 
-const Text = styled.div`
-  ${Fonts.garamond("s23")}
-`
+interface DivLayoutProps {
+  layout: String
+  postscript?: Boolean
+}
+
+const div: StyledFunction<DivLayoutProps & React.HTMLProps<HTMLDivElement>> = styled.div
 
+const Text = div`
+  position: relative;
+  padding-bottom: ${props => (props.postscript ? "2em" : null)};
+  a {
+    color: black;
+    text-decoration: none;
+    position: relative;
+    background-image: linear-gradient(to bottom,transparent 0,#333 1px,transparent 0);
+    background-size: 1.25px 3px;
+    background-repeat: repeat-x;
+    background-position: bottom;
+  }
+  p, ul, ol {
+    ${props => (props.layout === "classic" ? Fonts.garamond("s20") : Fonts.garamond("s23"))};
+    padding-top: ${props => (props.layout === "classic" ? ".75em;" : "1em;")};
+    padding-bottom: ${props => (props.layout === "classic" ? ".75em;" : "1em;")};
+    margin: 0;
+    font-style: ${props => (props.postscript ? "italic;" : "inherit;")};
```

Comment:
This should work: https://github.com/artsy/reaction/blob/master/src/components/publishing/fonts.tsx#L76